config/routes.rb

```diff
@@ -1,11 +1,16 @@
 Rails.application.routes.draw do
+  mount Rswag::Ui::Engine => '/api-docs'
+  mount Rswag::Api::Engine => '/api-docs'
+  # Defines the root path route ("/")
+  root "users#index"
+
+  devise_for :users
   # Define your application routes per the DSL in https://guides.rubyonrails.org/routing.html
+  
```

Comment:
- [ ]  kindly make you follow the requirements of the project by : 

1. Creating an API endpoint to list all posts for a user.
2. Creating an API endpoint to list all comments for a user's post.
3. Creating an API endpoint to add a comment to a post. Remember that the owner of the comment is the user that makes it; it should not be set by passing a parameter.
4. Your API endpoints should receive JSON and respond JSON as well.